src/CMakeLists.txt

```diff
@@ -123,7 +123,13 @@ set(THREADS_PREFER_PTHREAD_FLAG ON)
 
 ################################################################
 
+# abc does not have "platform" support for macos so force it to use stdint.h
+if (CMAKE_OSX_SYSROOT)
```

Comment:
I still think we should define the ABC_NAMESPACE define in order to get c++ namespaces.